.github/styles/Vocab/Base/accept.txt

```diff
@@ -135,4 +135,13 @@ bugfix
 nvm
 npm
 Gandi
-CrowdStrike
\ No newline at end of file
+CrowdStrike
+parallelization
+hyperscale
+goroutines
+arn
+arns
```

Comment:
`arns`? why? Shouldn't it always be `ARNs`?

Reply:
I think this was a merge artifact. I have removed it, lets see what breaks